src/index.js

```diff
@@ -26,9 +26,9 @@ function crossEnv(args) {
 
 function getCommandArgsAndEnvVars(args) {
   const envVars = getEnvVars()
-  const commandArgs = args.map(commandConvert)
-  const command = getCommand(commandArgs, envVars)
-  return [command, commandArgs, envVars]
+  const commandArgs = args.slice()
+  const command = commandConvert(getCommand(commandArgs, envVars))
+  return [commandConvert(command), commandArgs.map(commandConvert), envVars]
```

Comment:
Oh, I see. You are leveraging the fact that `getCommand` will strip env setters and the command from `commandArgs`. And you are making a copy so that `getCommandArgsAndEnvVars` does not itself mutate its argument. Am I correct? I think this is not very clear from the code. I understand it is done like this to minimize impact on existing code. It might still be a good idea to rename `getCommand` to something more revealing. However, the more I think about a name for this, the more I realize it does too much to have a good name:
- it strips env setters and the command from its first argument
- it sets env vars on its second argument
- it returns the command

A possible improvement would be to replace it with 2 functions:
- args -> [env setters, command, commandArgs] (simple to use using destructuring)
- env setters -> map of env vars (to be merged with `process.env` to form the env for the new process)

What do you think? Is it worth the work?